manifest.json

```diff
@@ -0,0 +1,246 @@
+        "version": "1.9.17.0",
+        "tarPath": "https://pypi.python.org/packages/source/G/GoogleAppEnginePipeline/GoogleAppEnginePipeline-1.9.17.0.tar.gz#md5=9fe87b281f4b0a7c110534df4e61b6ec",
+        "tarRootDirPrefix": "GoogleAppEnginePipeline-",
+        "rootDirPrefix": "gae-pipeline-",
+        "targetDirPrefix": "gae-pipeline-"
+      },
+      "graphy": {
+        "version": "1.0.0",
+        "tarPath": "https://pypi.python.org/packages/source/G/Graphy/Graphy-1.0.0.tar.gz#md5=390b4f9194d81d0590abac90c8b717e0",
+        "tarRootDirPrefix": "Graphy-",
+        "rootDirPrefix": "graphy-",
+        "targetDirPrefix": "graphy-"
+      },
+      "simpleJson": {
+        "version": "3.7.1",
+        "tarPath": "https://pypi.python.org/packages/source/s/simplejson/simplejson-3.7.1.tar.gz#md5=c76c2d11b87e9fb501bd0b2b72091653",
+        "tarRootDirPrefix": "simplejson-",
+        "rootDirPrefix": "simplejson-",
+        "targetDirPrefix": "simplejson-"
+      }
+    },
+    "thirdPartyStatic": {
+      "angular": {
+        "version": "1.4.7",
+        "filesPath": "https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7",
+        "targetDirPrefix": "angularjs-",
+        "files": [
+          "angular.js", "angular.min.js", "angular.min.js.map", "angular-animate.js",
+          "angular-animate.min.js", "angular-animate.min.js.map", "angular-resource.js",
+          "angular-resource.min.js", "angular-resource.min.js.map", "angular-route.js",
+          "angular-route.min.js", "angular-route.min.js.map", "angular-sanitize.js",
+          "angular-sanitize.min.js", "angular-sanitize.min.js.map", "angular-aria.js",
+          "angular-aria.min.js", "angular-aria.min.js.map"
+        ],
+        "jsFiles": [
+          "angular.js", "angular-animate.js", "angular-resource.js",
+          "angular-sanitize.js", "angular-aria.js"
+        ]
+      },
+      "angular-test": {
+        "version": "1.4.7",
+        "filesPath": "https://code.angularjs.org/1.4.7",
+        "targetDirPrefix": "angularjs-",
+        "files": ["angular-mocks.js", "angular-scenario.js"]
+      },
+      "D3": {
+        "version": "3.4.11",
+        "filesPath": "https://raw.github.com/mbostock/d3/v3.4.11",
+        "rootDirPrefix": "d3js-",
+        "targetDirPrefix": "d3js-",
+        "files": ["d3.min.js"],
+        "jsFiles": ["d3.min.js"]
+      },
+      "ngInfiniteScroll": {
+        "version": "1.0.0",
+        "filesPath": "https://raw.github.com/BinaryMuse/ngInfiniteScroll/1.0.0/build/",
+        "rootDirPrefix": "nginfinitescroll-",
+        "targetDirPrefix": "nginfinitescroll-",
+        "files": ["ng-infinite-scroll.min.js"],
+        "jsFiles": ["ng-infinite-scroll.min.js"]
+      },
+      "jquery": {
+        "version": "2.1.1",
+        "filesPath": "https://ajax.googleapis.com/ajax/libs/jquery/2.1.1",
+        "rootDirPrefix": "jquery-",
+        "targetDirPrefix": "jquery-",
+        "files": ["jquery.js", "jquery.min.js", "jquery.min.js.map"],
```

Comment:
So it looks like in some cases we actually download the minified files, which makes it seem a bit silly to minify them again...

Do all minified files have a .min.js or .min.css suffix? If so, maybe we should do something like this:
  - specify only the filenames (and not the extensions)
  - try to download .js and .min.js/.min.js.map into dev/prod respectively
    - if .min.js not found, minify the file ourselves on the fly and stick that into prod
  - ditto for css

Another approach is to indicate, in advance, that there's no min.js available by adding a boolean flag like "requiresMinification", which is a signal that we shouldn't try to fetch from those urls and instead just do our own.